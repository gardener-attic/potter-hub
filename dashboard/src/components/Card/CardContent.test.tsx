import { shallow } from "enzyme";
import * as React from "react";

import CardContent from "./CardContent";

it("should render the className", () => {
  const wrapper = shallow(<CardContent className="foo" />);
  expect(wrapper.find(".foo")).toExist();
});

it("should render the children elements", () => {
  const wrapper = shallow(
    <CardContent>
      <div>foo</div>
    </CardContent>,
  );
  expect(wrapper.text()).toContain("foo");
  expect(wrapper).toMatchSnapshot();
});
