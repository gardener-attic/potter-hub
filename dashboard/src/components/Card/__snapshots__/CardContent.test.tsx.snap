// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the children elements 1`] = `
<div
  className="Card__content padding-normal "
>
  <div>
    foo
  </div>
</div>
`;
